src/components/MunicipalCharts/ColumnChart.js

```diff
@@ -38,16 +38,21 @@ const humaniseRand = (x, longForm) => {
 }
 
 const formatter = (d, resultType) => {
-    if (resultType == 'currency') {
-        return humaniseRand(d, false);
-    } else if (resultType == 'months') {
-        return Math.round(d * 10) / 10;
-    } else if (resultType == 'percentage') {
-        return Math.round(d * 10) / 10 + '%';
-    } else if (resultType == 'ratio') {
-        return Math.round(d * 10) / 10;
-    } else {
-        return d;
+    if (d || d === 0 || d === -0) {
+        if (resultType == 'currency') {
```

Comment:
Using a `switch` would be cleaner here e.g.
```
switch(resultType) {
        case "currency": return humaniseRand(d, false);
        case "months": return Math.round(d * 10) / 10;
        case "percentage": return Math.round(d * 10) / 10 + "%";
        case "ratio": return Math.round(d * 10) / 10;
        default: return d;
    }
```

Reply:
Changed thanks